src/RiakClient.Tests.Live/BucketPropertyTests.cs

```diff
@@ -271,6 +271,29 @@ public void TestBucketTypesPropertyWorks()
             var plainBucketTypeBucketPropsResult = Client.GetBucketProperties("plain", "Schmoopy");
             plainBucketTypeBucketPropsResult.Value.DataType.ShouldBeNull();
         }
-    }
 
+        [Test]
+        public void TestConsistentPropertyOnRegularBucketType()
+        {
+            var regProps = Client.GetBucketProperties("plain", "bucket");
+            regProps.IsSuccess.ShouldBeTrue();
+            regProps.Value.Consistent.Value.ShouldBeFalse();
+        }
+
+        [Test]
+        public void TestConsistentPropertyIsNullOnNewProps()
+        {
+            var props = new RiakBucketProperties();
+            props.Consistent.HasValue.ShouldBeFalse();
+        }
+
+        [Test]
+        [Ignore("Test will only pass if you have a bucket type named \"consistent\" with SC enabled on it.")]
```

Comment:
Should be a new task to the integration test process ... set up a SC cluster with this bucket type.